### PR TITLE
Migrate ec2_eip* modules and tests

### PR DESCRIPTION
**Pull request by alinabuzachis**
Depends-On: https://github.com/ansible/zuul-config/pull/443

Migrate ec2_eip* modules and tests

### CODE REVIEW

File: tests/integration/targets/ec2_eip/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,1442 @@
+- name: Integration testing for ec2_eip
+  module_defaults:
+    group/aws:
+      aws_access_key: '{{ aws_access_key }}'
+      aws_secret_key: '{{ aws_secret_key }}'
+      security_token: '{{ security_token | default(omit) }}'
+      region: '{{ aws_region }}'
+    ec2_eip:
```
Comment (gravesm) [inline tests/integration/targets/ec2_eip/tasks/main.yml:8]: Same thing, it seems like this needs to be `amazon.aws.ec2.eip` now.

Comment (alinabuzachis) [reply tests/integration/targets/ec2_eip/tasks/main.yml]: Ah, done, thanks!

**Comment by gravesm** [comment]
recheck

**Review by abikouo — APPROVED** [review]

**Comment by alinabuzachis** [comment]
recheck

**Review by goneri — APPROVED** [review]

**Comment by alinabuzachis** [comment]
recheck

**Comment by goneri** [comment]
regate

**Pull request merged**